docs/modules/ROOT/pages/how-tos/deploy_ocp.adoc

```diff
@@ -0,0 +1,54 @@
+= Deploy on existing OpenShift cluster
+
+This guide describes how to deploy exoscale CCM on an existing cluster.
+
+== Step-by-step guide
+
+. Deploy component-exoscale-cloud-controller-manager
+
+. Patch infrastructure config
++
+IMPORTANT: This step triggers node reboots to apply the Kubelet flag `--cloud-provider=external`.
++
+[source,bash]
+----
+kubectl --as cluster-admin patch infrastructure.config  cluster --type=merge -p '{"spec":{"platformSpec":{"external":{"platformName":"exoscale"},"type":"External"}}}'
+----
++
+[source,bash]
+----
+kubectl proxy &
+curl -XPATCH -H"Content-Type: application/merge-patch+json" http://localhost:8001/apis/config.openshift.io/v1/infrastructures/cluster/status -d '{"status":{"platform":"External","platformStatus":{"external":{"cloudControllerManager":{"state":"External"}},"type":"External"}}}'
```

Comment:
recent-ish (alpha in 1.24, beta in 1.27, stable in 1.33) kubectl can patch `status` without the proxy+curl hack:

```suggestion
kubectl --as=cluster-admin patch infrastructure.config cluster --type=merge --subresource=status \
  -p  '{"status":{"platform":"External","platformStatus":{"external":{"cloudControllerManager":{"state":"External"}},"type":"External"}}}'
```